File: JavaScript/771. Jewels and Stones.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function(jewels, stones) {
    var a = jewels.split("");
    var b = stones.split("");
    var c = 0;
    a.forEach(elementa => {
        b.forEach(elementb => {
            if(elementa == elementb) c += 1;
        });
    });
    return c;
};

File: C++/771. Jewels and Stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int res = 0;

        for(char i: jewels){
            for(char j: stones){
                if(i == j) res++;
            }
        }

        return res;
    }
};
